app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, src_file, dst_file = command.split(" ")
```

Comment:
Code Efficiency: You correctly split the command into three parts, but there is no check if the command actually has three parts. If the command is incorrect and has less or more than three parts, your code will raise a ValueError. You should add a check for the length of the splitted command and handle this case appropriately.